pkg/gofr/datasource/opentsdb/opentsdb_test.go

```diff
@@ -2,11 +2,12 @@ package opentsdb
 
 import (
 	"context"
+	"crypto/rand"
```

Comment:
Why mot


```suggestion
	"crypto/rand/v2"
```

It's supposwd to be safer

Reply:
there is no crypto/rand/v2 package ..... I think you have confused it with math/rand package ...

Reply:
You didn't address this, right?

It's strange to see a crypto for generating a random value in a test

Reply:
Generating random values using crypto is much safer and more reliable and it generates truly random numbers when possible. But using it just for tests is up for debate. I would say either is fine

Reply:
I agree. Let's say it's OK